.github/actions/compress_sign_and_upload/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Compress and Sign
+description: 'Compresses package and signs with garasign'
+
+inputs: 
+    garasign_username:
+      description: 'Garasign username input for drivers-github-tools/garasign/gpg-sign'
+      required: true
+    garasign_password:
+      description: 'Garasign password input for drivers-github-tools/garasign/gpg-sign'
+      required: true
+    artifactory_username:
+      description: 'Artifactory username input for drivers-github-tools/garasign/gpg-sign'
+      required: true
+    artifactory_password:
+      description: 'Artifactory password input for drivers-github-tools/garasign/gpg-sign'
+      required: true
+
+runs:
+  using: composite
+  steps:
+    - run: npm pack
+      shell: bash
+
+    - name: Get release version and release package file name
```

Comment:
This looks generic enough where it's useful for other packages as well. We can consider extracting this into a separate action in the [Github Tools Repository](https://github.com/mongodb-labs/drivers-github-tools) for easier sharing across node repositories.

Reply:
yep, we'd just have to add a package_name field to the inputs of this file, and add it into `package_file`

(didn't realize I didn't send this comment, was in "pending" for 5 days)